microreactors/gcmr/core/ISOXML/Serpent/serpent_input

```diff
@@ -0,0 +1,732 @@
+ 6012.01c    1.0
+
+% --- Pyrolytic carbon layer:
+
+mat PyC2     -1.8820 moder grph 6012 tmp 725.0 rgb 0 200 200     
+ 6012.01c    1.0
+
+% --- Silicon carbide layer:
+
+mat SiC     -3.1710  tmp 725.0 rgb 0 100 0  
+14028.01c    0.4611
+14029.01c    0.0234
+14030.01c    0.0154
+ 6012.01c     0.5
+
+mat Nb   -8.4132  tmp 725.0   rgb 100 0 100   % Nobium 
+41093.01c  -0.99
+40090.01c  -5.132E-03
+40091.01c  -1.107E-03
+40092.01c  -1.674E-03
+40094.01c  -1.660E-03
+40096.01c  -2.618E-04
+
+mat Cr   -7.1900  tmp 725.0   rgb 100 0 200   % Chromium 
+24050.01c  -4.345e-2
+24052.01c  -83.789e-2
+24053.01c  -9.501e-2
+24054.01c  -2.365e-2
+
+% --- Graphite matrix:
+
+mat matrix_pin       -1.8060 moder grph 6012  tmp 725.0  rgb 220 220 220 
+ 6012.01c    1
+
+mat matrix       -1.8060 moder grph 6012  tmp 725.0 rgb 200 200 200      
+ 6012.01c  0.9999997
+ 5010.01c  0.0000003
+
+mat moderator   -4.0850 moder h-yh2 1001 moder y-yh2 39089 tmp 725.0   rgb 73 64 171
+39089.01c  0.357142857
+1001.01c   0.642857143
+
+mat Refl        -2.7987  moder beo 4009 tmp 725.0   rgb 100 220 200  % GREY
+ 4009.01c    0.5
+ 8016.01c    0.5
+
+% -----------------------------------------------------------------------------
+
+therm grph    grph.84t
+therm  beo     be-beo.84t
+therm h-yh2   h-yh2.84t
+therm y-yh2   y-yh2.84t
+
+% Calculation parameters:
+
+% -----------------------------------------------------------------------------
+
+% --- Geometry plot:
+plot 3 10000 10000   120.0
+plot 2 10000 10000   0.2
+
+% --- Libraries:
+
+set acelib "/home/ycao/XSLIB/serpent/sss_endfb8_serpent_yan.xsdir"
+set declib "/home/talamo/XSLIB/serpent/sss_endfb7.dec"
+set nfylib "/home/talamo/XSLIB/serpent/sss_endfb7.nfy"
+
+
+% --- Boundary conditions:
+
+set bc 1
+
+% --- Power density
+
+set power 20000000.0  % W
+
+% --- Histories:
+
+
+
+
+set pop 10000 300 30
+set mcvol 10000000000
+div fuel0 sep 4 
+div fuel1 sep 4 
+div fuel2 sep 4 
+div Gd_bpI1 sep 4 
+div Gd_bpI2 sep 4 
+div Gd_bpI3 sep 4 
+div Gd_bpO1 sep 4 
+div Gd_bpO2 sep 4 
+div Gd_bpO3 sep 4 
+
+% --- Nuclide inventory:
+dep daytot  1  4  30  90  150  240  390  540  690  840  990  1140  1290  1440  1590  1740  1890  2040  2190  2340  2490  2640  2880  3120  3360  3600
+
+set inventory
+
+% Actinides: 
+
+922350
+922380
+942390
+942400
+942410
+942420
+952410
+952440
+952450
+
+% Fission products:
+
+360850
+380900
+471101
+551370
+541350
+621490
+621510
+
+% burnable absorbents
+50100
+50110
+
+
+
+% 700 fuel0
+% 701 fuel1
+% 702 fuel2
+% 750-5    boron LBP0
+% 760-5    boron LBP1
+% 802 moderator  
+% 803    matrix  
+% 804    coolant 
+% 805   Axial Reflector
+% 806  shell_mod 
+% 807    coolant 
+% 809    boro control 
+% 810    Cr 
+% 811 Radial reflector   
+
+set nfg  11  8.00000e-08 1.80000e-07 6.25000e-07 1.30000e-06 4.00000e-06                                               
+             1.48730e-04 9.11800e-03 1.83000e-01 5.00000e-01 1.35300e+00                                               
+
```

Comment:
shouldn't we have a line to extract the XS in those universes for ISOXML generation? If so - it is important we specify it.